src/command_modules/azure-cli-resource/azure/cli/command_modules/resource/_params.py

```diff
@@ -59,9 +59,11 @@
 register_cli_argument('resource group', 'resource_group_name', resource_group_name_type, options_list=('--name', '-n'))
 register_cli_argument('resource group deployment', 'resource_group_name', arg_type=resource_group_name_type, completer=get_resource_group_completion_list)
 register_cli_argument('resource group deployment', 'deployment_name', options_list=('--name', '-n'), required=True, help='The deployment name.')
-register_cli_argument('resource group deployment', 'parameters_file_path', completer=FilesCompleter())
-register_cli_argument('resource group deployment', 'template_file_path', completer=FilesCompleter())
+register_cli_argument('resource group deployment', 'parameters_file_path', completer=FilesCompleter(), help='provide deployment parameter values, a file path, or an uri to a remote file, or json string')
```

Comment:
I think the key part is I still lean forward to support a regular file path without `@`, same with uri.